Automattic/wp-calypso
# user-devices: migrate to ReactQuery

#### Changes proposed in this Pull Request

* Migrate `user-devices` Redux state / data fetching to React Query

#### Testing instructions

* Go to `/me/notifications` with an account which as none, one or more devices connected
* Toggle settings for a site and verify the column regarding devices works exactly as on prod
* Go to `/me/notifications/comments` and verify settings are also working exactly as on prod
